ourjapanlife/findadoc-web
# Add Google Maps to MapContainer

This PR follows #267 

# What changed
1. Adds `runtimeConfig` to nuxt.config.js so the Google Maps API Key is loaded by Nuxt.
2. Adds the `vue3-google-map` package to implement Google Maps more easily.
3. Adds `<GoogleMap>` to the MapContainer to render Google Maps in the component.
4. Adds blue pin icon for map. Other icons can be added in later PRs.
5. Creates a locationStore as a temporary measure to handle the state for the location the user selects. Let's discuss how this might be better implemented in a future PR.
6. Updates the `SearchResultDetails` and `SearchResultsList` components to use computed values from Pinia.

# Testing instructions
Same testing instructions as #267. Only difference is that you should now see the mockData populate the modal and search results list.

# Screenshot
![Screenshot 2023-07-17 at 8 31 18 PM](https://github.com/ourjapanlife/findadoc-web/assets/31802656/ae025dfa-93a2-477f-ad1e-f9ac15a96495)
